modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -176,6 +166,29 @@ export default {
       this.unit = null;
       this.comment = '';
     },
+    async fetchCrops() {
+      const cropsResponse = await fetch(
+        'http://farmos/api/taxonomy_term/plant_type'
+      );
+      const crops = await cropsResponse.json();
+      this.cropList = crops.data;
+    },
+    async fetchPlantList(cropURL) {
+      const plantResponse = await fetch(cropURL);
+      const plants = await plantResponse.json();
+      this.plantList = plants;
+    },
+  },
+  watch: {
+    crop() {
+      console.log(this.crop.attributes.name);
+      this.fetchPlantList(
+        'http://farmos/api/fd2_plant_assets?crop=' + this.crop.attributes.name
+      );
+    },
+  },
+  async created() {
+    this.fetchCrops();
```

Comment:
The method you added to fetch the crop list is `async`.  So it should be called using `await` in your `created` lifecycle hook